gui/packages/ubuntupro/lib/pages/landscape/landscape_page.dart

```diff
@@ -0,0 +1,340 @@
+import 'dart:io';
+
+import 'package:file_picker/file_picker.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_markdown/flutter_markdown.dart';
+import 'package:provider/provider.dart';
+import 'package:ubuntu_service/ubuntu_service.dart';
+import 'package:wizard_router/wizard_router.dart';
+
+import '/core/agent_api_client.dart';
+import '/pages/widgets/page_widgets.dart';
+import 'landscape_model.dart';
+
+class LandscapePage extends StatelessWidget {
+  const LandscapePage({
+    super.key,
+    required this.onApplyConfig,
+    this.onSkip,
+    this.onBack,
+  });
+
+  final void Function() onApplyConfig;
+  final void Function()? onSkip;
+  final void Function()? onBack;
+
+  @override
+  Widget build(BuildContext context) {
+    final model = context.watch<LandscapeModel>();
+    final lang = AppLocalizations.of(context);
+    final linkStyle = MarkdownStyleSheet.fromTheme(
+      Theme.of(context).copyWith(
+        textTheme: DarkStyledLandingPage.textTheme.copyWith(
+          bodyMedium: DarkStyledLandingPage.textTheme.bodyMedium?.copyWith(
+            fontWeight: FontWeight.w100,
+          ),
+        ),
+      ),
+    ).copyWith(
+      a: const TextStyle(
+        decoration: TextDecoration.underline,
+      ),
+    );
+
+    return ColumnLandingPage(
+      svgAsset: 'assets/Landscape-tag.svg',
+      title: 'Landscape',
+      onNext: !model.hasError
+          ? () async {
+              if (await model.applyConfig() && context.mounted) {
+                onApplyConfig();
+              }
+            }
+          : null,
+      onBack: onBack ?? () => Wizard.of(context).back(),
+      onSkip: onSkip ?? () => Wizard.of(context).next(),
```

Comment:
Making these tearoffs causes tests to fail, since there is an error that the widget does not have a Wizard parent it leads to a bunch of errors constructing the widget. Maybe this is something we can address in our meeting.

Reply:
Yeah, that could be fixed if you put the LandscapePage under a Wizard in tests, but I agree that we start to depend on many external things just for testing, which is not that good. Let's leave as is, for the sake of simpler tests.